src/h5web/vis-packs/core/utils.ts

```diff
@@ -228,3 +234,9 @@ export function isScaleType(val: unknown): val is ScaleType {
     typeof val === 'string' && Object.values<string>(ScaleType).includes(val)
   );
 }
+
+function isSupported(val: number) {
+  const absVal = Math.abs(val);
+
+  return val === 0 || (absVal > Number.EPSILON && absVal < Number.MAX_VALUE);
```

Comment:
I had suggested `1 / Number.EPSILON` instead of `Number.MAX_VALUE`, but we could also use `Number.MAX_SAFE_INTEGER`. The problem with `Number.MAX_VALUE` is that you don't really know that you've gone above it, because there's no possible value above it. I suggested `1 / Number.EPSILON` for consistency, and because it gives a number about equal to about half of `MAX_SAFE_INTEGER`, which is still sufficiently large for our use case (I think).